webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/extractor/AssetPathToFilterConfigMapper.java

```diff
@@ -46,7 +48,8 @@
         MACSTRINGS(MACSTRINGS_FILTER_CONFIG_ID, "strings"),
         PO(POFilter.FILTER_CONFIG_ID, "pot"),
         RESX(RESX_FILTER_CONFIG_ID, "resx", "resw"),
-        XTB(XTB_FILTER_CONFIG_ID, "xtb");
+        XTB(XTB_FILTER_CONFIG_ID, "xtb"),
+        JS(JS_FILTER_CONFIG_ID, "ts", "js");
```

Comment:
and jsx files too?

Reply:
Do you know any teams using jsx file for string resources?